根模块中引入失败

问题：
在根模块中引入ElModule 失败

版本信息：
    " @ angular/animations": "^4.3.6",
    " @ angular/cdk": "^2.0.0-beta.8",
    " @ angular/cli": "^1.0.0",
    " @ angular/common": "^4.3.6",
    " @ angular/compiler": "^4.3.6",
    " @ angular/compiler-cli": "^4.3.6",
    " @ angular/core": "^4.3.6",
    " @ angular/forms": "^4.3.6",
    " @ angular/http": "^4.3.6",
    " @ angular/material": "^2.0.0-beta.8",
    " @ angular/platform-browser": "^4.3.6",
    " @ angular/platform-browser-dynamic": "^4.3.6",
    " @ angular/platform-server": "^4.3.6",
    " @ angular/router": "^4.3.6",
    " @ ngx-translate/core": "^6.0.1",
    " @ ngx-translate/http-loader": "^1.0.1",
    "angular-in-memory-web-api": "^0.3.2",
    "bootstrap": "^3.3.7",
    "bootstrap-table": "^1.11.2",
    "core-js": "^2.4.1",
    "echarts": "^3.5.3",
    "element-angular": "0.0.4-alpha.3",
    "font-awesome": "^4.7.0",
    "hammerjs": "^2.0.8",
    "jquery": "^3.2.1",
    "moment": "^2.18.1",
    "ng-zorro-antd": "^0.5.0",
    "ngx-bootstrap": "^1.6.6",
    "primeng": "^4.0.0-rc.3",
    "rxjs": "^5.1.0",
    "typescript": "^2.4.2",
    "zone.js": "^0.8.4"

`app.module.ts`中的代码：
```typescript
import { ElModule } from 'element-angular'
import 'element-theme-default';
//import other modules

@NgModule({
  declarations: [
    AppComponent
  ],
  imports: [
    ElModule.forRoot(),
    BrowserModule,
   //other modules
  ],
  providers: [],
  bootstrap: [AppComponent]
})
export class AppModule { }
```

错误信息：
ERROR in /root/Documents/magicCarFinder/magiccarfinderclient/src/app/app.module.ts (13,26): Cannot find module 'element-angular'.
ERROR in Error: Error encountered resolving symbol values statically. Could not resolve element-angular relative to /root/Documents/magicCarFinder/magiccarfinderclient/src/app/app.module.ts., resolving symbol AppModule in /root/Documents/magicCarFinder/magiccarfinderclient/src/app/app.module.ts, resolving symbol AppModule in /root/Documents/magicCarFinder/magiccarfinderclient/src/app/app.module.ts, resolving symbol AppModule in /root/Documents/magicCarFinder/magiccarfinderclient/src/app/app.module.ts
    at Error (native)
    at syntaxError (/root/Documents/magicCarFinder/magiccarfinderclient/node_modules/@angular/compiler/bundles/compiler.umd.js:1725:34)
    at simplifyInContext (/root/Documents/magicCarFinder/magiccarfinderclient/node_modules/@angular/compiler/bundles/compiler.umd.js:24949:23)
    at StaticReflector.simplify (/root/Documents/magicCarFinder/magiccarfinderclient/node_modules/@angular/compiler/bundles/compiler.umd.js:24961:13)
    at StaticReflector.annotations (/root/Documents/magicCarFinder/magiccarfinderclient/node_modules/@angular/compiler/bundles/compiler.umd.js:24391:41)
    at _getNgModuleMetadata (/root/Documents/magicCarFinder/magiccarfinderclient/node_modules/@angular/compiler-cli/src/ngtools_impl.js:138:31)
    at _extractLazyRoutesFromStaticModule (/root/Documents/magicCarFinder/magiccarfinderclient/node_modules/@angular/compiler-cli/src/ngtools_impl.js:109:26)
    at Object.listLazyRoutesOfModule (/root/Documents/magicCarFinder/magiccarfinderclient/node_modules/@angular/compiler-cli/src/ngtools_impl.js:53:22)
    at Function.NgTools_InternalApi_NG_2.listLazyRoutes (/root/Documents/magicCarFinder/magiccarfinderclient/node_modules/@angular/compiler-cli/src/ngtools_api.js:91:39)
    at AotPlugin._getLazyRoutesFromNgtools (/root/Documents/magicCarFinder/magiccarfinderclient/node_modules/@ngtools/webpack/src/plugin.js:207:44)
    at _donePromise.Promise.resolve.then.then.then.then.then (/root/Documents/magicCarFinder/magiccarfinderclient/node_modules/@ngtools/webpack/src/plugin.js:443:24)
    at process._tickCallback (internal/process/next_tick.js:103:7)

--- a/ex/app/components/guide/install/code.ts
+++ b/ex/app/components/guide/install/code.ts
@@ -5,12 +5,12 @@ export default [
 
 `,
 
-// 引入
+// 引入 webpack
 `
 // import module
 import { ElModule } from 'element-angular'
 
-// import style
+// if you use webpack, import style
 import 'element-theme-default'
 
 @NgModule({
@@ -24,6 +24,17 @@ import 'element-theme-default'
 
 // components
 template: '<el-button>Hello World</el-button>'
+
+`,
+
+// angular-cli
+`
+// in /src/styles.css
+// add code:
+
+/* You can add global styles to this file, and also import other style files */
+@import "~element-theme-default";
+
 
 `,
 

--- a/ex/app/components/guide/install/install.component.html
+++ b/ex/app/components/guide/install/install.component.html
@@ -7,11 +7,15 @@
 
 <h3>Hello world</h3>
 <p>引入模块后，我们能轻易地构建一个简易页面</p>
-<p><code>element-theme-default</code> 样式库是 <code>Element Angul</code> 的依赖，无需再次安装。</p>
+<p><code> element-theme-default </code> 样式库是 <code>Element Angular</code> 的依赖，无需再次安装。</p>
 
-<p>注: 假设你已经通过 <a href="https://github.com/angular/angular-cli" target="_blank">angular-cli</a> 创建了基础的 Angular 项目。</p>
+<p>使用 webpack 的引入方式：</p>
 <pre>
   <code class="hljs" [innerHTML]="code[1] | highlightAuto"></code>
+</pre>
+<p>如果你使用 angular-cli，你需要样式文件在 <code>/src/styles</code> 引入：</p>
+<pre>
+  <code class="hljs" [innerHTML]="code[2] | highlightAuto"></code>
 </pre>
 
 

--- a/ex/docs/changelog.json
+++ b/ex/docs/changelog.json
@@ -1,81 +1,92 @@
 {
   "releases-link": "https://www.npmjs.com/package/element-angular",
   "releases": [
-  {
-    "version": "0.0.1",
-    "date": "2017-08-06",
-    "link": null,
-    "events": [{ "description": "创建项目" }]
-  },
-  {
-    "version": "0.0.3-alpha.2",
-    "date": "2017-08-30",
-    "link": null,
-    "events": [{ "description": "完成主要组件。补充组件文档以及安装引导。" }]
-  },
-  {
-    "version": "0.0.3-alpha.3",
-    "date": "2017-09-02",
-    "link": null,
-    "events": [{
-      "description": "修复 Aot 编译警告"
-    }, {
-      "description": "增加 Notification / message 组件"
-    }]
-  },
-  {
-    "version": "0.0.4-alpha.0",
-    "date": "2017-09-03",
-    "link": null,
-    "events": [{
-    }, {
-      "description": "增加 Cascader / Card / Badge / Dropdown 组件与文档"
-    }, {
-      "description": "修复 Input / Button 组件的继承样式问题"
-    }]
-  },
-  {
-    "version": "0.0.4-alpha.1",
-    "date": "2017-09-06",
-    "link": null,
-    "events": [{
-      "description": "增加 DatePicker / Breadcrumb 组件与文档"
-    }, {
-      "description": "增加 Input 图标事件与文档",
-      "issues": []
-    }, {
-      "description": "增加 Dropdown 自动改变方向",
-      "issues": [143]
-    }, {
-      "description": "修复 Select 禁用仍旧可以选中",
-      "issues": [136]
-    }]
-  },
-  {
-    "version": "0.0.4-alpha.2",
-    "date": "2017-09-08",
-    "link": null,
-    "events": [{
-      "description": "增加 Dialog / Carousel / Collapse / Slider 组件与文档"
-    }, {
-      "description": "增加 Progress 自定义颜色",
-      "issues": [140]
-    }, {
-      "description": "修复 Select 清除后未更新列表激活项",
-      "issues": [135]
-    }, {
-      "description": "修复 Tag 文档缺失",
-      "issues": [139]
-    }]
-  },
-  {
-    "version": "0.0.4-alpha.3",
-    "date": "2017-09-12",
-    "link": null,
-    "events": [{
-      "description": "增加 Alert / Menu / Pagination 组件与文档"
-    }]
-  }
+    {
+      "version": "0.0.1",
+      "date": "2017-08-06",
+      "link": null,
+      "events": [{ "description": "创建项目" }]
+    },
+    {
+      "version": "0.0.3-alpha.2",
+      "date": "2017-08-30",
+      "link": null,
+      "events": [{ "description": "完成主要组件。补充组件文档以及安装引导。" }]
+    },
+    {
+      "version": "0.0.3-alpha.3",
+      "date": "2017-09-02",
+      "link": null,
+      "events": [{
+        "description": "修复 Aot 编译警告"
+      }, {
+        "description": "增加 Notification / message 组件"
+      }]
+    },
+    {
+      "version": "0.0.4-alpha.0",
+      "date": "2017-09-03",
+      "link": null,
+      "events": [{
+      }, {
+        "description": "增加 Cascader / Card / Badge / Dropdown 组件与文档"
+      }, {
+        "description": "修复 Input / Button 组件的继承样式问题"
+      }]
+    },
+    {
+      "version": "0.0.4-alpha.1",
+      "date": "2017-09-06",
+      "link": null,
+      "events": [{
+        "description": "增加 DatePicker / Breadcrumb 组件与文档"
+      }, {
+        "description": "增加 Input 图标事件与文档",
+        "issues": []
+      }, {
+        "description": "增加 Dropdown 自动改变方向",
+        "issues": [143]
+      }, {
+        "description": "修复 Select 禁用仍旧可以选中",
+        "issues": [136]
+      }]
+    },
+    {
+      "version": "0.0.4-alpha.2",
+      "date": "2017-09-08",
+      "link": null,
+      "events": [{
+        "description": "增加 Dialog / Carousel / Collapse / Slider 组件与文档"
+      }, {
+        "description": "增加 Progress 自定义颜色",
+        "issues": [140]
+      }, {
+        "description": "修复 Select 清除后未更新列表激活项",
+        "issues": [135]
+      }, {
+        "description": "修复 Tag 文档缺失",
+        "issues": [139]
+      }]
+    },
+    {
+      "version": "0.0.4-alpha.3",
+      "date": "2017-09-12",
+      "link": null,
+      "events": [{
+        "description": "增加 Alert / Menu / Pagination 组件与文档"
+      }]
+    },
+    {
+      "version": "0.0.4-alpha.4",
+      "date": "2017-09-14",
+      "link": null,
+      "events": [{
+        "description": "修复 Module build 的错误",
+        "issues": [178]
+      }, {
+        "description": "争对 Angular Cli 项目完善文档"
+      }]
+    }
   ]
 }
 

--- a/index.ts
+++ b/index.ts
@@ -1,0 +1,1 @@
+export * from './release/index'

--- a/package.json
+++ b/package.json
@@ -1,11 +1,12 @@
 {
   "name": "element-angular",
-  "version": "0.0.4-alpha.3",
+  "version": "0.0.4-alpha.4",
   "description": "element ui for angular",
-  "main": "release/index.d.ts",
+  "main": "./index.ts",
   "unpkg": "bundle/element-angular.js",
   "files": [
     "release",
+    "index.ts",
     "bundle"
   ],
   "scripts": {
@@ -80,7 +81,8 @@
     "zone.js": "^0.8.16"
   },
   "peerDependencies": {
-    "@angular/core": "^4.3.4"
+    "@angular/core": "^4.0.0",
+    "@angular/animations": "^4.0.0"
   },
   "dependencies": {
     "element-theme-default": "^1.4.4"

